### ISSUE TITLE
Future of this soundfont

### ISSUE THREAD

**Issue by Kappa971** [trevor0402/SC55Soundfont#3]
Hi and first of all congratulations for the soundfont, among those I have tried, it is the one that comes closest to a Roland sc-55. However this also has problems with some instruments, for example the Descent 1 menu music is reproduced in a "strange" way compared to how it should. Are there any new versions of this soundfont planned in the future?

**Comment by Kappa971** [comment]
One of the developers of the tool used to convert the SC-55 rom into soundfont, clearly stated that, if he were to continue the project, he would opt to create an emulator based on MUNT (contacting the developers) or create a new soundfont format that supports all features of the SC-55 (which the .sf2 format cannot do). So I guess this soundfont will never be 100% accurate...

**Comment by itsragedev** [comment]
> One of the developers of the tool used to convert the SC-55 rom into soundfont ... or create a new soundfont format that supports all features of the SC-55

Sounds cool imo, do you know what its name will be? (maybe like sf3 or something)

**Comment by Kappa971** [comment]
I have no idea, you have to ask the developer

**Comment by RavenMacDaddy** [comment]
Apparently FluidSynth supports the SF3 spec; https://www.fluidsynth.org/

More info on this format specifically can be found on its wiki here: https://github.com/FluidSynth/fluidsynth/wiki/SoundFont3Format

**Comment by Kappa971** [comment]
> Apparently FluidSynth supports the SF3 spec; https://www.fluidsynth.org/
> 
> More info on this format specifically can be found on its wiki here: https://github.com/FluidSynth/fluidsynth/wiki/SoundFont3Format

Interesting. You may open a problem here https://github.com/Kitrinx/SC55_Soundfont, maybe the developer (the one who created the tool to convert SC-55 roms to sf2 soundfont) might be interested.

**Comment by Kappa971** [comment]
A new project was born: https://github.com/skjelten/emusc

This is what the developer wrote in Vogons:
_Hi everyone,_

_Inspired by the excellent work you guys have done to decode the control and wave ROMs I have spent a month to see if I was able to write a simple emulator that would use the original ROM files to recreate the sounds of my SC-55. Considering the fact that I had not programmed for 10+ years and knew little about synthesizers it's been a pretty steep learning curve. Luckily the SC55 sound font by Kitrinx and NewRisingSun was extremely helpful._

_The project is naturally in very early stages of development and currently limited to Linux & ALSA, but I believe I have gotten the basic parts to work. Next step is to figure out some kind of RIAA filtering and adding ADSR/volume envelope. Feel free to check it out at https://github.com/skjelten/emusc_

_Some of you might wonder why I did not just fork munt, but I found that to be an overwhelming task. I needed to start from scratch to understand all the inner workings of both the ROMs and how a synth works._

_PS: Any help would be greatly appreciated!_
